code/modules/surgery/bodyparts/ghetto_parts.dm

```diff
@@ -0,0 +1,79 @@
+/obj/item/bodypart/arm/left/ghetto
+	name = "left peg arm"
+	desc = "A roughly hewn wooden peg replaces where a forearm should be. It's simple and sturdy, clearly made in a hurry with whatever materials were at hand. Despite its crude appearance, it gets the job done."
+	icon = 'icons/mob/human/species/ghetto.dmi'
+	icon_static = 'icons/mob/human/species/ghetto.dmi'
+	limb_id = BODYPART_ID_PEG
+	icon_state = "peg_l_arm"
+	bodytype = BODYTYPE_PEG
+	should_draw_greyscale = FALSE
+	attack_verb_simple = list("bashed", "slashed")
+	unarmed_damage_low = 3
+	unarmed_damage_high = 9
+	unarmed_effectiveness = 5
+	brute_modifier = 1.2
+	burn_modifier = 1.5
+	bodypart_traits = list(TRAIT_CHUNKYFINGERS) // TRAIT_CHUNKYFINGERS is stupid as shit and will be borked within a week, but by god i'm using it (please merge b4 it gets changed so its not my problem)
```

Comment:
Don't put very quickly outdated temporal comments in your code.

Reply:
fixed